fdbclient/include/fdbclient/FDBTypes.h

```diff
@@ -946,6 +946,27 @@ struct KeyValueStoreType {
 			return "unknown";
 		}
 	}
+
+	// Convert a string to a KeyValueStoreType
+	// This is a many-to-one mapping as there are aliases for some storage engines
+	static KeyValueStoreType fromString(const std::string& str) {
```

Comment:
Since this isn't a performance relevant function ()and therefore doesn't need to be inlined) it should go into a cpp file. This file is included almost everywhere and this will increase compilation time.

Reply:
Done.